src/web/tools/InteractionManager.ts

```diff
@@ -103,14 +102,15 @@ export default class InteractionManager {
   }
 
   public shouldHandlerBeCancelledBy(
-    _handler: GestureHandler,
-    otherHandler: GestureHandler
+    _handler: IGestureHandler,
+    otherHandler: IGestureHandler
   ): boolean {
-    return (
-      otherHandler instanceof NativeViewGestureHandler &&
-      otherHandler.getState() === State.ACTIVE &&
-      !otherHandler.isButton()
-    );
+    const isNativeHandler =
```

Comment:
Please add a comment why it's done this way

Reply:
Added in [89337e2](https://github.com/software-mansion/react-native-gesture-handler/pull/2783/commits/89337e2553cb74ab31602a4295fa4281cebdb9e0).